api/tests/unit/application/organizations/organization-controller_test.js

```diff
@@ -1012,4 +1013,25 @@ describe('Unit | Application | Organizations | organization-controller', functio
       expect(response.headers['Content-Disposition']).to.contains('attachment; filename=20210101_attestations_3b.pdf');
     });
   });
+
+  describe('#getOrganizationMembers', function () {
+    it('should return the firstName and lastName of all members of the organization', async function () {
```

Comment:
suggestion `should return all members of the organization with only firstName and lastName` pour mettre l'accent sur le fait que ça renvoie la liste des membres/prescripteurs en prio

Reply:
je remplace par `should return all members of the organization serialized`